packages/portfolio-contract/src/plan-solve.ts

```diff
@@ -373,10 +387,11 @@ export const solveRebalance = async (
   return flows;
 };
 
-export const rebalanceMinCostFlowSteps = (
+export const rebalanceMinCostFlowSteps = async (
   flows: SolvedEdgeFlow[],
   graph: RebalanceGraph,
-): MovementDesc[] => {
+  gasEstimator?: GasEstimator,
```

Comment:
This parameter should be required.
```suggestion
  gasEstimator: GasEstimator,
```

Reply:
this code is also being used `multichain-testing/scripts/ymax-tool.ts` through the `makePortfolioSteps` function. there no gasEstimator is present.